meps/mep-cross-references.md

```diff
@@ -0,0 +1,360 @@
+| Local file download      | `<path:file.txt>`          | `[](file.txt)`            |
+| File download (explicit) | `<path:file.md>`           | `[](path:file.md)`        |
+| Project document         | `<project:file.md>`        | `[](file.md)`             |
+| Target in a document     | `<project:target.md#file>` | `[](file.md#target)`      |
+| Target in project        | `<project:#target>`        | `[](#target)`             |
+
+### Search Order and Specificity
+
+All references search the local document first[^specific_doc], then the local project in the order of the table of contents. A `xref_ambiguous` warning is raised if multiple matches are found.
+
+[^specific_doc]: With the exception of an explicit reference to a specific page, i.e. `[](./examples/my-doc.md#explicit-reference)`
+
+In large documentation sites, a referenced target can be present in multiple documents, in that case, the parser will emit a `xref_ambiguous` warning letting you know that there are multiple matches for the intended target.
+If a link cannot be resolved, an external link should be rendered, for example, `<a href="#target">#target</a>`.
+
+### Implicit Section Headers
+
+We suggest a configuration option to create anchor "slugs" for section headers, which stay close to the [GitHub implementation](https://github.com/Flet/github-slugger), which produces references that:
+
+- lower-case text
+- remove punctuation
+- replace spaces with `-`
+- enforce uniqueness via suffix enumeration `-1`
+
+For example, `## Links and Referencing` can be referenced as `[](#links-and-referencing)`.
+Every heading level in a document should have an anchor, however, these are **implicit** references, and referring to them can raise an `xref_implicit` warning, which can optionally be suppressed by users.
+
+Implicit references are **not** available project wide, and are only accessible in the current document, as many documents follow similar structures (Abstract, Introduction, Methods, Summary).
+Adding two sections of the same name does not raise a duplicate identifier warnings (`xref_duplicate`), section identifiers are only unique to the document.
+
+### Paths
+
+- Links can be relative from the containing file.
+- Links can be absolute paths from the project root.
+- Absolute paths start with `/`.
+- The path separator is POSIX `/`.
+- The path must include the extension.
+
+### Downloads
+
+Files that are outside of the table of contents of the project and are referenced directly are downloads.
+
+- These follow the same path rules as above when referring to a unknown file-type (e.g. `./my-file.txt`).
+- Files with known file extensions (e.g. `*.md`, `*.ipynb`) will default to being document links, not downloads.
+- Downloads can be specified explicitly by a `<path:my-file.md>`, which will revert to a download regardless of the extension. (i.e. this overrides the default for `[](my-file.md)`, which is `<project:my-file.md>`).
```

Comment:
An implementation note: `path` was chosen over, the perhaps more obvious `file`, because markdown-it (and possibly other parsers?) explicitly block `file` schema, to prevent XSS attacks

Reply:
oops no I see this down in the bottom section